_episodes/14-writing-functions.md

```diff
@@ -77,17 +77,14 @@ print_date(1871, 3, 19)
 ~~~
 {: .output}
 
-*   Via [Twitter](https://twitter.com/minisciencegirl/status/693486088963272705):
```

Comment:
I like this analogy and would rather is not be removed from the lesson.  Is there a specific reason you think it should be cut?